operations/helm/charts/mimir-distributed/templates/gateway/gateway-ingress.yaml

```diff
@@ -1,9 +1,6 @@
 {{- if .Values.enterprise.enabled -}}
 {{- if .Values.gateway.ingress.enabled -}}
-{{- $ingressApiIsStable := eq (include "mimir.ingress.isStable" .) "true" -}}
-{{- $ingressSupportsIngressClassName := eq (include "mimir.ingress.supportsIngressClassName" .) "true" -}}
-{{- $ingressSupportsPathType := eq (include "mimir.ingress.supportsPathType" .) "true" -}}
-apiVersion: {{ include "mimir.ingress.apiVersion" . }}
+apiVersion: networking.k8s.io/v1
```

Comment:
Let's add a CHANGELOG entry `[CHANGE]` to highlight that ingress will not render on pre 1.19 kubernetes.

Reply:
addresseed in https://github.com/grafana/mimir/pull/2872/commits/f793324039e2460480df7205916e6236b0da4df8